cmd/krew/cmd/install.go

```diff
@@ -91,7 +91,7 @@ Remarks:
 
 			var install []index.Plugin
 			for _, name := range pluginNames {
-				plugin, err := indexscanner.LoadPluginByName(paths.IndexPluginsPath(), name)
+				plugin, err := indexscanner.LoadPluginByName(paths.IndexPluginsPath(""), name)
```

Comment:
to  reinforce my previous comment, you might as well use `"default"` here. Later on, you'll have to scan every single usage of these methods and replace them all over again. Using `"default"` can be make your job as it's easier to find.